elements/pl-card/pl-card.mustache

```diff
@@ -0,0 +1,31 @@
+<div class="card w-{{width}} my-4">
+  {{#img-top-src}}
+  <img src="{{img-top-src}}" class="card-img-top">
+  {{/img-top-src}}
+  {{#header}}
+  <div class="card-header">
+    {{header}}
```

Comment:
I think we probably want to avoid using indentation in the odd case someone passes components with `markdown`. (Unless the lambda in `pl-card.py` handles that.)

Reply:
Markdown would have been turned into HMTL markup long before we get here.